katsdpimager/frontend.py

```diff
@@ -279,9 +266,14 @@ def log_parameters(self, suffix=''):
         log_parameters("CLEAN parameters" + suffix, self.clean_p)
 
 
+def prepend_dunder(value):
```

Comment:
'dunder' always makes me chuckle:) Strictly speaking should you call a '--' a dunder? I always thought it was '__'.

Reply:
Good point! I've renamed to `prepend_dashes`.